### PR TITLE
Updated error-handling sections

### PR DESCRIPTION
**Pull request by sarah-mason**
* Created a section called Errors that fail transfers
* Made edits to other common behaviours (including removing transfers that belonged in the previous section)
* All checked section against style guide

This is connected to issue: https://github.com/archivematica/Issues/issues/569

### CODE REVIEW

File: getting-started/troubleshooting/error-handling.rst
Comment (sallain) [inline getting-started/troubleshooting/error-handling.rst]: This is a very minor thing, but an unordered list might be better than an ordered list here? I totally leave this up to you though.

File: getting-started/troubleshooting/error-handling.rst
Comment (sallain) [inline getting-started/troubleshooting/error-handling.rst:269]:
```suggestion
Below is a list of common errors that will produce an
```
It could be confusing to refer to normalization here, if someone hasn't read the normalization section above.

File: getting-started/troubleshooting/error-handling.rst
Comment (sallain) [inline getting-started/troubleshooting/error-handling.rst:313]:
```suggestion
It is recommended that you :ref:`clean up the Transfer tab <cleanup>` and :ref:`the Ingest tab <cleanup-ingest>`
```

File: getting-started/troubleshooting/error-handling.rst
Comment (sallain) [inline getting-started/troubleshooting/error-handling.rst:323]:
```suggestion
transfers in the dashboard (`Issue 7246`_). Minimal, but successful, testing
```
Ooooh I thought I'd caught all the period+doublespace occurrences in the docs but every once in a while I find one!

**Review by sallain — CHANGES_REQUESTED** [review]
A few small recommendations, but otherwise this looks good to me!

File: getting-started/troubleshooting/error-handling.rst
Comment (sarah-mason) [reply getting-started/troubleshooting/error-handling.rst:269]: good point!

File: getting-started/troubleshooting/error-handling.rst
Comment (sarah-mason) [reply getting-started/troubleshooting/error-handling.rst]: valid point. It saves future edits from making sure they remain in order.

**Comment by sarah-mason** [comment]
don't mind the two commits. The first one wouldn't push to the branch and in the meantime I made another change. For some reason when I pushed that one it worked and both went through.

**Review by sallain — APPROVED** [review]

**Comment by sallain** [comment]
@sarah-mason So that one Travis check is still failing, but it's because it's comparing against a specific, older commit - probably the commit that the 1.10 branch was at the last time you pushed. I've since fixed the TOC entry that was causing the error, but I don't know how to make Travis look at the correct commit. I think you should be good to merge - let me know if you aren't allowed.

**Comment by ross-spencer** [comment]
Hi @sallain @sarah-mason to take into account changes to the TOC in upgrading.rst in a previous commit you need to rebase to bring this branch back in line to remove that warning.

You just need to be on this development branch locally, and then:
```
git fetch --all && git rebase origin/1.10
git push -f
```
And if you want before the git push, you can test with:
```
make html
```
Where it will complain beforehand, and not afterwards. 

This process will update the branch here and build correctly. I'm sure there's an updated version of [this](https://wiki.archivematica.org/Merging#Merge_process) somewhere? But it's step 3, replacing `qa/1.x` with `1.10` here because that's your target for the docs repo. 

I suspect you maybe right though Sara (trying to remember my git-fu), it might not matter when working on the individual file, you're not likely to overwrite anything, but that's the risk we're mitigating by following that step. 

**NB.** For Sarah M's sake if you haven't come across it, we try to never force push on a main branch as it will overwrite things irreversibly. We only do it on dev branches where we're trying to bring a dev branch in-line with main.

**Comment by sallain** [comment]
@ross-spencer Thanks! @sarah-mason if you want to try following Ross' instructions, hopefully that will clear up that last pesky failure.

As a sidenote, I think the only time that the docs docs tell you to force-push is when you're updating a dev branch, which should be fine?

As a second sidenote, I always run `make clean html` rather than `make html` because the latter will only catch errors on files you've changed, whereas the former will check all files. Really helpful for a situation like this, where the error was caused by someone else merging with errors!

**Comment by sarah-mason** [comment]
Thanks @sallain and @ross-spencer! Handy tips in there that I will try before I merge this morning. I knew yesterday when I ran `make clean html` I was still getting the error from the previous errors merged. I spent a fair bit of time reading about rebasing and testing various things to get it work properly (hence the two commits in a row), but I hadn't found these instructions! 🤞

**Comment by sarah-mason** [comment]
Nice @ross-spencer -- that `git fetch` actually worked! So nice to finally see all green! 👍 

And thanks for all the explanations re. force pushing and what the instructions do. I seemed to have had a better time with rebasing by using `git fetch` rather than `git pull`. It may have had something to do with all the merges happening around this and the opaque nature of `git pull`.

**Pull request merged**
